src/test/resources/pacts/publicapi-direct-debit-connector-search-mandates-endpoint.json

```diff
@@ -0,0 +1,220 @@
+{
+  "consumer": {
+    "name": "publicapi"
+  },
+  "provider": {
+    "name": "connector"
+  },
+  "interactions": [
+    {
+      "description": "search mandates by parameters",
+      "providerStates": [
+        {
+          "name": "a gateway account with an external ids and a confirmed mandate with an email exists",
+          "params": {
+            "gateway_account_id": "2po9ycynwq8yxdgg2qwq9e9qpyrtre",
+            "mandate_id": "jkdjsvd8f78ffkwfek2a",
+            "unique_identifier": "AAAAA",
+            "charge_id": "charge97837509646393e3A",
+            "email": "rory@example.com"
+          }
+        }
+      ],
+      "request": {
+        "method": "GET",
+        "path": "/v1/api/accounts/123456/mandates",
+        "query": {
+          "email": [
```

Comment:
why is the email value in an array out of interest?

Reply:
I thought this was how params worked in pact from looking at other tests, they had single value array members

Reply:
I'm no expert, looks like you can use an array to pass a list of values...
https://docs.pact.io/getting_started/matching#query-params